src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,76 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { useAppDispatch } from '../../app/hooks';
+import { getUser } from '../../api';
+import { actions } from '../../features/currentTodo';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
+type Props = {
+  selectedTodo: Todo,
+};
 
-      <Loader />
+export const TodoModal: React.FC<Props> = ({ selectedTodo }) => {
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+  const dispatch = useAppDispatch();
 
-      <div className="modal-card">
-        <header className="modal-card-head">
-          <div
-            className="modal-card-title has-text-weight-medium"
-            data-cy="modal-header"
-          >
-            Todo #3
-          </div>
+  useEffect(() => {
+    if (selectedTodo) {
```

Comment:
Consider handling possible errors when fetching the user data with a .catch() block.